apidoc/Titanium/UI/View.yml

```diff
@@ -1033,7 +1033,7 @@ properties:
   - name: anchorPoint
     summary: Coordinate of the view about which to pivot an animation.
     description: |
-        Used on iOS only. For Android, use <Titanium.UI.Animation.anchorPoint>. 
+        Used on iOS only. For Android, use pivotX and pivotY. 
```

Comment:
Why not calling it `anchorPoint` on Android as well then, making those ones `x` and `y` ? :-) #parity 🔥